src/exercise/02.md

```diff
@@ -2,6 +2,8 @@
 
 ## 📝 Your Notes
 
+- Interestingly, it requires a space to be added to match the expected.
```

Comment:
i thought this was strange too! perhaps the space could be included in one of the variables?

Reply:
I initially did that, but then the spans don't appear like they do in the example solution. I think from a practical perspective, it really doesn't "matter", but from a CSS perspective, it might make sense _not_ to wrap the space with your span.